packages/playwright/src/reporter.ts

```diff
@@ -328,9 +329,19 @@ class ReplayPlaywrightReporter implements Reporter {
     try {
       await this.reporter.onEnd();
       if (!this._foundReplayBrowser) {
```

Comment:
wouldn't this prevent the installation warning from being logged when projects are configured correctly but our browser is not installed?

Reply:
I don't understand what you mean.

I didn't change the naming of `_foundReplayBrowser` but the way it was meant to be used was to flag whether or not Replay Chromium was in one of the projects that ran. That's separate from whether or not it's actually installed. (I check for that below using `!existsSync(getRuntimePath())`)

Reply:
That's what I tried to say - both are kinda separate problems but the `existsSync` check is within the `! this._foundReplayBrowser` branch.

So after merging this, if I **did** configure the project to run with `replay-chromium` but forgot to actually install the Replay Chromium I end up with this output:
```
  ✘  1 [replay-chromium] › test/example.spec.ts:3:5 › basic test (8ms)


  1) [replay-chromium] › test/example.spec.ts:3:5 › basic test ─────────────────────────────────────

    Error: browserType.launch: Failed to launch chromium because executable doesn't exist at /Users/mateuszburzynskiprev/.replay/runtimes/Replay-Chromium.app/Contents/MacOS/Chromium

  1 failed
    [replay-chromium] › test/example.spec.ts:3:5 › basic test ──────────────────────────────────────
[replay.io]: 🕑 Completing some outstanding work ...
[replay.io]: 
```

Reply:
I see what you're saying. I didn't change that behavior in this PR, but it was a scenario we weren't calling out before.

Reply:
I have filed [PRO-751](https://linear.app/replay/issue/PRO-751) to follow up on this and will tackle it once you land #585